lib/generators/decidim/install_generator.rb

```diff
@@ -38,7 +38,14 @@ def copy_migrations
       end
 
       def add_seeds
-        append_file "db/seeds.rb", "Decidim::Core::Engine.load_seed"
+        append_file "db/seeds.rb" do
+          "\n# Loads seeds from all engines.
+Rails.application.railties.select do |railtie|
```

Comment:
Not sure about this approach. Aren't we triggering seeding from potentially a lot of engines? Why doesn't rails do this automatically when running `rake db:seed`? 

Maybe we could provide a `Decidim.seed` method that does all that, while checking the module `Decidim` is prepended to the engine's name. This way we de-clutter all this and make it more scalable (imagine we change the implementation of this seeding thinguie).